crates/db_schema/src/impls/tagline.rs

```diff
@@ -0,0 +1,54 @@
+use crate::{
+  source::tagline::{Tagline, TaglineInsertForm, TaglineUpdateForm},
+  traits::Crud,
+};
+use diesel::{dsl::*, result::Error, PgConnection, QueryDsl, RunQueryDsl};
+
+impl Crud for Tagline {
+  type InsertForm = TaglineInsertForm;
```

Comment:
Check out the `FederationAllowList` forms. There's really only a need to clear, and update.